modules/va_facilities/app/controllers/va_facilities/v1/nearby_controller.rb

```diff
@@ -51,6 +52,10 @@ def set_default_format
         request.format = :json if params[:format].nil? && request.headers['HTTP_ACCEPT'].nil?
       end
 
+      def set_facility_type
+        params[:type] = 'health'
```

Comment:
what do you think about making this name consistent with the types in BaseFacility, for example: "va_health_facility"